pkg/collector/corechecks/containers/docker.go

```diff
@@ -42,7 +42,7 @@ type DockerConfig struct {
 	CollectImageSize         bool               `yaml:"collect_image_size"`
 	CollectDiskStats         bool               `yaml:"collect_disk_stats"`
 	CollectVolumeCount       bool               `yaml:"collect_volume_count"`
-	Tags                     []string           `yaml:"tags"`
+	Tags                     []string           `yaml:"tags"` // Used only by the configuration converter v5 → v6
```

Comment:
Keeping this attribute only because it's used by the legacy converter is not great. But for now, I've no good solution except this comment to avoid somebody to use this field.

Reply:
If you want to be bullet-proof, you can use anonymous embedding to create a new type extending DockerConfig, in the legacy package:

```
type fullDockerConfig struct {
    containers.DockerConfig
    Tags                     []string           `yaml:"tags"`
}
```

But your approach is OK